hoon/reference/stdlib/2b.md

```diff
@@ -564,6 +564,11 @@ the final value of the accumulator.
 (To "slam" means to call a gate and give it a sample/samples. In this instance,
 `a` is the list of samples that are given to the gate b.)
 
+The initial value of the accumulator is `+>+.$`. This can occasionally produce
+unexpected behavior (see examples). If you need more control over the initial
```

Comment:
```suggestion
undesired behavior (see examples). If you need more control over the initial
```

It's not really unexpected if you've read these docs and/or know what's going on. And it's the _undesired_ behavior you always want to affect, not necessarily the unexpected. ...But maybe that's too pedantic.